watertap/property_models/anaerobic_digestion/tests/test_modified_adm1_reaction.py

```diff
@@ -595,13 +591,13 @@ def model(self):
         m.fs.unit.inlet.conc_mass_comp[0, "X_ch"].fix(7.3687)
         m.fs.unit.inlet.conc_mass_comp[0, "X_pr"].fix(7.7308)
         m.fs.unit.inlet.conc_mass_comp[0, "X_li"].fix(10.3288)
-        m.fs.unit.inlet.conc_mass_comp[0, "X_su"].fix(1e-6)
-        m.fs.unit.inlet.conc_mass_comp[0, "X_aa"].fix(1e-6)
-        m.fs.unit.inlet.conc_mass_comp[0, "X_fa"].fix(1e-6)
-        m.fs.unit.inlet.conc_mass_comp[0, "X_c4"].fix(1e-6)
-        m.fs.unit.inlet.conc_mass_comp[0, "X_pro"].fix(1e-6)
-        m.fs.unit.inlet.conc_mass_comp[0, "X_ac"].fix(1e-6)
-        m.fs.unit.inlet.conc_mass_comp[0, "X_h2"].fix(1e-6)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_su"].fix(1e-8)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_aa"].fix(1e-8)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_fa"].fix(1e-8)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_c4"].fix(1e-8)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_pro"].fix(1e-8)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_ac"].fix(1e-8)
+        m.fs.unit.inlet.conc_mass_comp[0, "X_h2"].fix(1e-8)
```

Comment:
curious why 0s were able to be set for other model specifications after changes but not here

Reply:
That change affected the translators